zeek/pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
```

Comment:
```suggestion
```

We don't need setuptools. fixing this in template here: https://github.com/DataDog/integrations-core/pull/16527